dags/glam_subdags/daily_aggregates.py

```diff
@@ -0,0 +1,114 @@
+from airflow import DAG
+
+from utils.gcp import gke_command, bigquery_xcom_query
+
+dags = {}
+
+
+def generate_tasks_for_clients_daily(
+    metric_type, agg_type, dataset_id, dag, arguments=()
+):
+    gen_query_task_id = "clients_daily_{}_aggregates_generate_query".format(agg_type)
+
+    # setting xcom_push to True outputs this query to an xcom
+    generate_query = gke_command(
+        task_id=gen_query_task_id,
+        command=[
+            "python",
+            "sql/telemetry_derived/clients_daily_{}_aggregates_v1.sql.py".format(
+                metric_type
+            ),
+            "--agg-type",
+            agg_type,
+        ],
+        docker_image="mozilla/bigquery-etl:latest",
+        xcom_push=True,
+        owner="msamuel@mozilla.com",
+        email=[
+            "telemetry-alerts@mozilla.com",
+            "msamuel@mozilla.com",
+            "robhudson@mozilla.com",
+        ],
+        dag=dags[dag],
+    )
+
+    run_query = bigquery_xcom_query(
+        task_id="clients_daily_{}_aggregates_run_query".format(agg_type),
+        destination_table="clients_daily_{}_aggregates_v1".format(metric_type),
+        dataset_id=dataset_id,
+        xcom_task_id=gen_query_task_id,
+        owner="msamuel@mozilla.com",
+        email=[
+            "telemetry-alerts@mozilla.com",
+            "msamuel@mozilla.com",
+            "robhudson@mozilla.com",
+        ],
+        arguments=arguments,
+        dag=dags[dag],
+    )
+    generate_query >> run_query
+    return run_query
+
+
+def daily_scalar_aggregates_subdag(
+    parent_dag_name, child_dag_name, default_args, schedule_interval, dataset_id
+):
+    GLAM_DAILY_SCALAR_AGGREGATES_SUBDAG = "%s.%s" % (parent_dag_name, child_dag_name)
```

Comment:
Or as an f-string:

`     GLAM_DAILY_SCALAR_AGGREGATES_SUBDAG = f"{parent_dag_name}.{child_dag_name}"

Reply:
Airflow is still on python 2.7, so no f-strings yet.